packages/shared/src/table/selectable-table.tsx

```diff
@@ -18,6 +18,12 @@ import { useSortList } from '../hooks/sort-list';
 import { getUID } from '../selectors';
 import { useCustomTranslation } from '../useCustomTranslationHook';
 
+export type TableColumnProps = ThProps & {
+  thProps?: TableThProps;
+  columnName: string;
+  sortFunction?: (a: any, b: any, c: SortByDirection) => any;
```

Comment:
```suggestion
  sortFunction?: (a: any, b: any, c: SortByDirection) => any;
```
Can we add types for these?